tinker/templates/admin/redirects/home.html

```diff
@@ -164,12 +164,35 @@
                     input.val("/" + value); // add in a slash
                     value = input.val();
                 }
+                if (value[0] == '/' && value[1] =='/'){
+                    input.val(input.val().replace('//','/')); // eliminates a lead double slash
+                }
                 delay(function () {
                     loadTableForSearch(value, $('#new-redirect-to').val());
                     $('#spinner').hide();
                 }, 300);
             });
 
+            $('#modal-from-path-edit').keyup(function(){
```

Comment:
didn't we already have keyup functions? Or am i remembering wrong...

Reply:
We didn't... I think I meant to put them in but I must have gotten distracted by something else and forgot. Better late than never??